buildtools/log4j.xml

```diff
@@ -3,32 +3,16 @@
 $Id$
 -->
 
-<!DOCTYPE log4j:configuration SYSTEM "log4j.dtd">
-
-<log4j:configuration xmlns:log4j="http://jakarta.apache.org/log4j/"
-                     debug="false">
-
-  <!-- ============================================== -->
-  <!-- Appenders                                      -->
-  <!-- ============================================== -->
-
-  <appender name="CONSOLE" class="org.apache.log4j.ConsoleAppender">
-    <errorHandler class="org.apache.log4j.helpers.OnlyOnceErrorHandler"/>
-    <param name="Target" value="System.out"/>
-    <param name="Threshold" value="WARN"/>
-  </appender>
-
-  <!-- ============================================== -->
-  <!-- Categories                                     -->
-  <!-- ============================================== -->
-
-
-  <!-- ============================================== -->
-  <!-- The Root Category                              -->
-  <!-- ============================================== -->
-
-  <root>
-    <priority value="ALL"/>
-    <appender-ref ref="CONSOLE"/>
-  </root>
-</log4j:configuration>
+<Configuration>
+  <Appenders>
+    <Console name="CONSOLE" target="SYSTEM_OUT" ignoreExceptions="false">
+      <ThresholdFilter level="WARN"/>
+    </Console>
+  </Appenders>
+  
+  <Loggers>
+    <Root level="debug">
+      <AppenderRef ref="STDOUT"/>
```

Comment:
Shouldn't the reference be`ref="CONSOLE"` ?

Reply:
Updated to `ref="CONSOLE"`